consensus/spos/interface.go

```diff
@@ -170,3 +172,9 @@ type PeerBlackListCacher interface {
 	Sweep()
 	IsInterfaceNil() bool
 }
+
+// EquivalentMessagesDebugger defines the specific debugger for equivalent messages
+type EquivalentMessagesDebugger interface {
+	DisplayEquivalentMessagesStatistics(getDataHandler func() map[string]uint64)
```

Comment:
let's make the signature more straightforward. Why not declare the function like
```GO
DisplayEquivalentMessagesStatistics(messages map[string]uint64)
```